static/templates/settings/data_exports_admin.hbs

```diff
@@ -28,7 +28,9 @@
         </div>
     </form>
     {{/if}}
-    <div class="admin-table-wrapper">
+    <input type="text" class="search" placeholder="{{t 'Filter exports' }}"
+      aria-label="{{t 'Filter exports' }}"/>
+    <div class="progressive-table-wrapper" data-simplebar data-list-render="admin_exports_list">
```

Comment:
@rishig did you intentionally remove this "filter" widget in your latest batch of changes?  I could go either way on whether we want it; it's sorta good for consistency but also only likely to be useful if someone has scripted frequent exports through this code path (which is totally possible).  Probably my preference would be no "filter" widget.

Reply:
oops, not intentional, but also agree that we can leave it out.

Reply:
Cool, just take it out then?